RFCs/005-DynamicData-Attribute.md

```diff
@@ -0,0 +1,51 @@
+# RFC 005- DynamicData Attribute for Data Driven Tests
+
+## Summary
+This details the MSTest V2 framework attribute "DynamicData" for data driven tests where data can be specified as property.
+
+## Motivation
+Often times, the inline test data need to be shared between tests. There could be more than one test cases in different files which will we using same test data. In most of these situations, user end up declaring same inline data for multiple test cases using "DataRow" attribute. Test framework should provides an attribute so that test data, declared as property, can be
+* Easily re-used
+* Easily organized
+
+## Detailed Design
+
+### Requirements
+1. Test data declared as property should be easily asscoaited with test case.
```

Comment:
should this actually be - "Inline test data can be reused by multiple test cases"? That would cover both the requirements you have.